assets/i18n/qqq.json

```diff
@@ -1,4 +1,41 @@
 {
+	"I18N_ABOUT_PAGE_ABOUT_TAB_CREATE": "Text shown at the bottom of the About section of Oppia's About page.",
+	"I18N_ABOUT_PAGE_ABOUT_TAB_CREATE_TEXT": "Text shown at the bottom of the About section of Oppia's About page.",
```

Comment:
I think you might want to provide some info to translators (who may not actually see the Oppia website) of how this message relates to the previous one. It's not clear that it is a continuation. I also wonder how well this construct would work in other languages.